docs/source/vllm_integration.md

```diff
@@ -185,3 +185,12 @@ Given these factors, our experiments on the Qwen model family (3B, 7B, 14B, 32B)
 
 * For reasonable-sized models (3B–14B) and a moderate context window (`max_len < 8k`), using full capacity for data parallelism gives better throughput. The setup `(tp=1, dp=8)` yields the best results.
 * For larger models (32B) and longer context windows (`max_len > 8k`), a smaller DP size combined with some model-side parallelism performs better. For example, `(tp=2, dp=4)` is a good setup for 32B models with a larger context window.
+
+## vLLM with Transformers Backend
+
+vLLM now supports transformers backend for model implementations. Simply passing in `transformers` in `vllm_model_impl` in configurations or through argument parser will set use transformers backend. See an example below.
```

Comment:
Does it support VLMs? Limitations? 
Additionally, maybe linking the blog: https://blog.vllm.ai/2025/04/11/transformers-backend.html
(in case these ideas can be added in 1-2 sentences)